job_templates/ci_job.xml.em

```diff
@@ -133,6 +133,10 @@ if [ "$CI_USE_OPENSPLICE" = "true" ]; then
 fi
 if [ -n "${CI_ROS2_REPOS_URL+x}" ]; then
   export CI_ARGS="$CI_ARGS --repo-file-url $CI_ROS2_REPOS_URL"
+@[ if turtlebot_demo ]@
```

Comment:
As a temporary measure, this change is scoped to the turtlebot_demo job.
Once it tests fine there, I'll update this so it's not scoped to
turtlebot and add the same thing to the Windows bits.